src/mgmt/srv_drpc.c

```diff
@@ -59,10 +59,10 @@ ds_mgmt_drpc_prep_shutdown(Drpc__Call *drpc_req, Drpc__Response *drpc_resp)
 	D_INFO("Received request to prep shutdown %u\n", req->rank);
 
 #ifndef DRPC_TEST
-	ds_pool_disable_evict();
+	ds_pool_disable_exclude();
```

Comment:
This only disable local swim DEAD event callback.  Do you remember why do we need it?